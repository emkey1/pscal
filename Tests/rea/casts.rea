writeln(int(3.7));
writeln(int(-3.7));
writeln(int(float(5)));
writeln(int(float(6)));
writeln(int(char(300)));
writeln(int(bool(0.0)));
writeln(int(bool(2)));
