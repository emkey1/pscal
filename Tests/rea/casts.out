3
-3
5
6
44
0
1
